wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -272,4 +276,10 @@ public void setScreenOrientation(ScreenOrientation orientation) throws Exception
     public void unsetScreenOrientation() throws Exception {
         getUiDevice().unfreezeRotation();
     }
+
+   public void uiDevicePerformLongClick(UiObject view, int steps) throws Exception {
```

Comment:
Also we should start adding comments to explain what a step value really equates to make the code more easily understood.